CMakeLists.txt

```diff
@@ -608,6 +608,8 @@ IF (SONAME)
 	IF (LIBGIT2_FILENAME)
```

Comment:
When using `-DLIBGIT2_FILENAME=git2.dll`, the resulting file will be `libgit2.dll.dll`.